docs/conversion/s2u_features/S2U-14-oracle.sql

```diff
@@ -0,0 +1,3 @@
+-- S2U-14 --
+ALTER TABLE SAM_PUBLISHEDITEM_T ADD CANCELLED NUMBER(10,0) DEFAULT 0 NOT NULL;
```

Comment:
Seems too much for me to store something smaller than 100, do you know if number(2,0) is enough?